dockerfiles/Dockerfile.konflux

```diff
@@ -117,17 +117,15 @@ COPY --from=test-image /is_tested /is_tested
 # STAGE 7 - PyPI publish package
 ###############################################################################
 FROM test-image AS pypi
-ARG TWINE_USERNAME
-ARG TWINE_PASSWORD
 
 # Lets make sure we ran previous prod stage before uploading to pypi
 COPY --from=prod-image /is_tested /is_tested
 RUN echo "true" > /is_pypi_pushed
 
 # qontract-reconcile version depends on git tags!
-# The .git dir should already be part of the test image.
-# TODO
-# RUN make pypi
+# The .git dir should already be part of the test image
+# TODO: re-enable to take over from ci.ext
+#RUN --mount=type=secret,id=app-sre-pypi-credentials/token export UV_PUBLISH_TOKEN=$(cat /run/secrets/app-sre-pypi-credentials/token); make pypi-konflux
```

Comment:
can just be
`RUN --mount=type=secret,id=app-sre-pypi-credentials/token,env=UV_PUBLISH_TOKEN make pypi-konflux`

Reply:
`env` is not supported bu buildah

Reply:
😿